.travis.yml

```diff
@@ -2,12 +2,21 @@ language: python
 python:
   - "2.6"
   - "2.7"
+services:
+  - "riak"
```

Comment:
I see we've added a MessageStore -- that seems like a crazy requirement but I'm going to keep reading.